docs/lib/auth/fragments/flutter/user_attributes/10_fetch_attributes.md

```diff
@@ -1,6 +1,9 @@
 ```dart
 try {
   var res = await Amplify.Auth.fetchUserAttributes();
+  res.forEach((element) {
```

Comment:
*Note:* I added this just for consistency with the docs below (as well as iOS/Android) where the result is printed.